project-management.md

```diff
@@ -0,0 +1,35 @@
+# Project Management
+
+The OpenTelemetry has limited bandwith for managing changes which expand the scope of OpenTelemetry, or impact many SIGs within OpenTelemetry. There are three common scenarios which have this kind of impact.
+* Non-trivial changes to the OpenTelemetry specification.
+* A new SIG being formed.
+* An existing SIG taking on new work which will affect the OpenTelemetry as a whole, and will need review from the broader community.
+
+Any changes which fall into one of the above categories must first create a project proposal and gain approval from the GC and TC before work begins.
+
+## Project Proposal
+
+At minimum, projects require the following resources to be successful:
+
+* A clearly defined set of goals and deliverables.
+* Deadlines for when the deliverables will be ready for review by the broader community.
```

Comment:
Deadlines do sound strict, but I think it will help the working groups focus and move towards delivering on their goals faster. Some things I have noticed when organizing prior projects:

* Having clear deadlines makes it easier for project members to receive approval to work on the project. When the work is open-ended, it is harder to get an approval from managers to use work time. This leads to engineers getting overburdened or pulled off the project. This has been particularly true for subject matter experts, who are participating in a particular project but are not planning on being a long term contributor to OpenTelemetry.
* Without a deadline, projects can become prone to "architectural astronauting," where they endlessly debate alternatives. Deadlines provide project leaders with a basis for ending debate and getting the group to produce a public proposal.
* Deadlines allow participants in the public review process to manage their calendars. Proper review can actually take a lot of time on the part of reviewers. Without deadlines, the public review period comes as a surprise, which leads to drawn out process due to critical participants being unavailable and unable to coordinate a response. This can be very frustrating to the project, as they are blocked on continuing their work until TC members and other approvers complete their review.

Obviously, a project can misjudge the amount of time they need, and can run into unexpected problems. In those cases, there is nothing wrong with updating the deadlines to something more realistic. But I think it is important that projects always maintain a clear expectation for when they will be delivering work for public review.

Reply:
Sounds good, thanks.